### DISCUSSION TITLE
Update Next.js foundations course to use app router

### DISCUSSION THREAD

**Discussion by belugawhale3** [vercel/next.js#54996] [App Router]
https://nextjs.org/learn/foundations/about-nextjs?utm_source=next-site&utm_medium=navbar&utm_campaign=home

Could we update the Nextjs foundations course to use app router? It currently uses page system and needs updating.

**Comment by icyJoseph** [comment]
I do believe Lee Robinson is working on this: https://github.com/vercel/next.js/issues/53271#issuecomment-1664600813

**Reply by belugawhale3** [reply]
Any word on when they might release their update?
